app/javascript/packs/dashboard.js

```diff
@@ -1,7 +1,5 @@
 import * as dashboardWidget from 'Dashboard/ajax-widget'
 import { render as renderChartWidget } from 'Dashboard/chart'
 
-document.addEventListener('DOMContentLoaded', () => {
-  window.dashboardWidget = dashboardWidget
-  window.renderChartWidget = renderChartWidget
-})
+window.dashboardWidget = dashboardWidget
+window.renderChartWidget = renderChartWidget
```

Comment:
If these exist before content was loaded, then maybe fix makes sense. But I'll let UI experts chime in.